iceoryx_examples/iceperf/CMakeLists.txt

```diff
@@ -16,6 +16,8 @@
 cmake_minimum_required(VERSION 3.5)
 project(example_iceperf)
 
+option(TOML_CONFIG "Activates or deactivates TOML support - without TOML RouDi will not be build" ON)
```

Comment:
We should probably resolve this dependency. I suppose it will only possible with v.2.0 and not v1.x Mentioned it in #230

Reply:
well, it's quite easy to build RouDi with a static config instead of a TOML config. The question is just if we want to do it if the TOML feature is disabled.

Reply:
We build in CI now both. With and without TOML config to ensure compatibility.

Reply:
The question I intended to ask was more like if we want do build a default RouDi even if TOML is not used?